src/Network/Network/VirtualNetwork/NewAzureVirtualNetworkCommand.cs

```diff
@@ -125,6 +125,12 @@ public class NewAzureVirtualNetworkCommand : VirtualNetworkBaseCmdlet
             HelpMessage = "The edge zone of the virtual network.")]
         public string EdgeZone { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "The PrivateEndpointVNetPolicies of the virtual network")]
+        public string PrivateEndpointVNetPoliciesValue { get; set; }
+
```

Comment:
Please help the help doc for the cmdlet. Refer to https://github.com/Azure/azure-powershell/blob/main/documentation/development-docs/help-generation.md